### ISSUE TITLE
Fix screen brightness

### ISSUE THREAD

**Issue by hockeymikey** [hockeymikey/android_device_sony_lilac#33]
https://github.com/hockeymikey/android_device_sony_lilac/blob/oreo/overlay/frameworks/base/core/res/res/values/config.xml

Lower it and maybe add a setting too.

https://forum.xda-developers.com/t/guide-enhanced-brightness-save-battery.2367412/

Just fix the levels. Maybe add a min setting later

**Comment by AM4Ryzen** [comment]
Yes, this ROM has the brightest minimum brightness that I've seen.

**Comment by AM4Ryzen** [comment]
Do you have a lowered file that I can test?

**Comment by hockeymikey** [comment]
Can try this one: https://raw.githubusercontent.com/hockeymikey/android_device_sony_lilac/oreo/overlay/frameworks/base/core/res/res/values/config.xml
put it in /system/overlay/frameworks/base/core/res/res/values/config.xml
Let me know if that works. Wish I had more time to update stuff, just been so busy in my life.

**Comment by AM4Ryzen** [comment]
I can't seem to find a overlay folder in system, but next time I use my phone, I'll check for it in TWRP, if I remember.

**Comment by AM4Ryzen** [comment]
There is no overlay folder in system that I can find.

**Comment by hockeymikey** [comment]
Yeah try making it manually

**Comment by AM4Ryzen** [comment]
> Yeah try making it manually

Ok, but it doesn't appear to have changed the screen brightness.

**Comment by hockeymikey** [comment]
https://forum.xda-developers.com/t/guide-enhanced-brightness-save-battery.2367412/

**Comment by AM4Ryzen** [comment]
Does this mean new version soon?

**Comment by hockeymikey** [comment]
@asdf2345 Eventually. I gotta build it first and dealing with some OS related issues with Arch + kde. And I need to fix my main xz1c since I'm currently using the backup and will wanna test on there.

**Comment by the-r3dacted** [comment]
Too lazy to log into my old account, so you get my new one.

I figured out how to change the brightness, I manually added this commit to framework-res.apk and it seems to be fixed.

https://github.com/whatawurst/android_device_sony_lilac/commit/844b62f254c9cd4dca43837c11f071c40dcb038b

[framework-res.zip](https://github.com/user-attachments/files/16443502/framework-res.zip)
